datastore/datastore_manager.go

```diff
@@ -49,18 +56,32 @@ func (d *Manager) CleanupRecords(ctx context.Context, valid []string) error {
 	}
 	q := datastore.NewQuery(kindName)
 	var records []*StoredMetricRecord
-	if _, err := q.GetAll(ctx, &records); err != nil {
+	if _, err := d.Client.GetAll(ctx, q, &records); err != nil {
 		return fmt.Errorf("could not list metric records: %v", err)
 	}
-	log.Infof(ctx, "%d metrics configured, %d metric records found in Datastore", len(valid), len(records))
+	log.WithContext(ctx).Infof("%d metrics configured, %d metric records found in Datastore", len(valid), len(records))
 	for _, r := range records {
 		if !existing[r.Name] {
-			log.Infof(ctx, "deleting obsolete metric record for %s", r.Name)
-			err := datastore.Delete(ctx, r.key(ctx))
+			log.WithContext(ctx).Infof("deleting obsolete metric record for %s", r.Name)
+			err := d.Client.Delete(ctx, r.key(ctx))
 			if err != nil {
 				return fmt.Errorf("could not delete metric record %v: %v", r.Name, err)
 			}
 		}
 	}
 	return nil
 }
+
+// Datastore doesn't need to be closed, this function exists for compatibility
```

Comment:
Function comments must start with the name of the function.

`go vet` or `golint` should pick this up for you.

Reply:
Done. Added automatic golint run to my IDE.